dashboard/app/models/script_level.rb

```diff
@@ -66,6 +66,48 @@ def stage_total
     stage.script_levels.to_a.count
   end
 
+  def summarize
+    if level.unplugged?
+      kind = 'unplugged'
+    elsif assessment
+      kind = 'assessment'
+    else
+      kind = 'puzzle'
+    end
+
+    summary = {
+        id: level.id,
```

Comment:
It's confusing for the `summarize` ID to actually refer to the `Level` record ID.